src/routes/Search.svelte

```diff
@@ -0,0 +1,98 @@
+<script lang="ts">
+  let currentInput: string = "";
+
+  //Likely replace later with full lessons from ts file
+  const lessons: { title: string; link: string }[] = [
+    { title: "One Variable Linear Equations", link: "/linear-equations" },
+    { title: "Pythagorean Theorem", link: "/pytheorem" },
+    { title: "Polynomials", link: "/polynomials" },
+  ];
+
+  let lessonsSearchedFor: { title: string; link: string }[] = [];
+
+  const updateSearch = () => {
+    if (currentInput.trim() === "") {
+      lessonsSearchedFor = [];
+    } else {
+      lessonsSearchedFor = lessons.filter((lesson) =>
+        lesson.title.toLowerCase().includes(currentInput.toLowerCase())
```

Comment:
I will have to create an API for this. Maybe connect tags to topics/units/lessons and search for them.

Reply:
Updated, see if that was what you wanted